src/components/edit/event/EventDialog.js

```diff
@@ -348,9 +348,7 @@ export class EventDialog extends Component {
                                         onChange={setEventPointColor}
                                         style={styles.flexInnerColumn}
                                     />
-                                    <TextField
-                                        id="radius"
```

Comment:
`id` not needed anymore?

Reply:
No, it might have been used in a cypress test, but I presume we can use dataTest moving forward.